core/src/test/java/org/apache/accumulo/core/conf/AccumuloConfigurationTest.java

```diff
@@ -394,42 +386,17 @@ public void testScanExecutors() {
 
   // note: this is hard to test if there aren't any deprecated properties
   // if that's the case, just comment this test out or create a dummy deprecated property
-  @SuppressWarnings("deprecation")
   @Test
   public void testResolveDeprecated() {
     var conf = new ConfigurationCopy();
 
-    // deprecated first argument
-    var e1 = assertThrows(IllegalArgumentException.class, () -> conf
-        .resolve(Property.INSTANCE_DFS_DIR, Property.INSTANCE_DFS_URI, Property.INSTANCE_DFS_URI));
-    assertEquals("Unexpected deprecated INSTANCE_DFS_DIR", e1.getMessage());
-
```

Comment:
Some of these tests are testing behavior of the `resolve` method. We may want to leave some kind of stub in place of these. Without a deprecated property to test resolution with, we can't perform these tests... but we will probably want to perform them if/when we deprecate properties in the future.

Reply:
I wonder if it is better to try and create some fake properties for the tests or something to check the resolve method vs leaving in stubs for non existent properties. Seems like if we want to check the resolve method we should create a test class specifically for that.

Reply:
Ideally, we would want to create test properties that aren't mixed in with the other properties... so we don't have to filter them out everywhere we enumerate real properties. If the method can be made generic in some way, perhaps something like `public <T> T resolve(Enumeration<T> .... propEnums);`, then it might be easy to create a test enum class for testing the method. However, the method does depend on being able to read the properties from the AccumuloConfiguration to see if they are set there, so it's not immediately obvious to me how we'd make that work without making it super convoluted.

Reply:
So I came up with the idea to take two properties (I just picked two random ones) and just mark them as deprecated just for the test using reflection and then restore them at the end. This way we didn't have to do any other magic and the test will always work going forward. It seemed like the simplest/cleanest solution in this case. It's a little hacky to have to use reflection to do this but I think it's ok in this case since it's just a unit test. Let me know what you think. I also fixed/updated my PR based on your other comments.

Reply:
I should also add I considered trying to use mocking and use EasyMock to mock the enum values but enums are final so you can't use normal mocking with EasyMock. I haven't used it before but I think the only way it could work was something like PowerMock but I checked and it is only set up for Junit 4 and not Junit 5. So it seemed like this was a simpler solution than trying to figure out a way to use PowerMock/EasyMock as getting that set up seems out of scope of this PR and if we wanted to do that should be another Issue/PR.

Reply:
clever! I like it